pkg/operator/controller/status.go

```diff
@@ -15,24 +13,23 @@ import (
 
 	"github.com/sirupsen/logrus"
 
-	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
 
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
-
-	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 const (
-	DNSClusterOperatorName = "dns"
+	DNSOperatorName     = "dns"
+	OperatorVersionName = "operator"
+	DNSVersionName      = DNSOperatorName
```

Comment:
This is no longer used in this PR, but no harm in leaving it in since #92 will most likely use it.